screengrab/lib/screengrab/runner.rb

```diff
@@ -230,6 +230,15 @@ def pull_screenshots_from_device(device_serial, device_screenshots_paths, device
 
       UI.verbose("Starting screenshot count is: #{starting_screenshot_count}")
 
+      # adb creates a different directory structure depending on whether or not the destination directory
+      # exists before the pull. Work around this by ensuring that the destination directory always exists.
+      #
+      # See: https://github.com/fastlane/fastlane/pull/4915#issuecomment-236368649
+      unless File.exist?(@config[:output_directory])
```

Comment:
Yep, true! I actually started off writing it as `File.directory?`, but changed it last minute. I can change it back if you prefer.